### ISSUE TITLE
Edit Schema

### ISSUE THREAD

**Issue by agegendo** [json-editor/json-editor#890]
### General information 
- json-editor version: 2.5.1

### Question
Hi, 

I've tried to figure out if it's possible to edit the JSON schema itself with this editor but couldn't figure out a way to do it. The goal would be to allow a user to add new properties and that these properties are reflected in an updated schema that can be saved (similar to how it's done in [this project](https://bjdash.github.io/JSON-Schema-Builder/)).

So basically I want to know if it's possible to get an updated schema if new properties are added.

Thanks!

**Comment by schmunk42** [comment]
You can build schemas with https://github.com/json-editor/json-editor/blob/master/tests/pages/meta_schema.json
